Commands/Public/contacts.js

```diff
@@ -199,7 +199,14 @@ module.exports = async(client, msg, suffix) => {
 				msg.author.busy = false;
 				return omsg.delete().catch(e => null);
 			}
-
+			if (/^8$/.test(collected.first().content)) {
+				collected.first = contact.name;
+			}
+			if (collected.first.length() < 0 || collected.first.length() > 21) {
+				let isItMax = collected.first.length() > 21;
+				msg.author.busy = false;
+				return msg.channel.send({ embed: { title: `Description ${isItMax ? "too long" : "too short"}`, description: `Please ${isItMax ? "lengthen" : "shorten"} the description.` } });
```

Comment:
Don't return, this will be very annoying. Turn it into a feedback loop.

Reply:
that is planned (it might take some time tho)